src/components/FilmCard/FilmCard.js

```diff
@@ -1,8 +1,8 @@
-import React, { Component } from "react";
-import { Link } from "react-router-dom";
-import Button from "@material-ui/core/Button";
-import "./style.scss";
-import { slugify } from "transliteration";
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import Button from '@material-ui/core/Button';
+import './style.scss';
+import { slugify } from 'transliteration';
```

Comment:
You don't need to import slugify anymore. I think you can also remove it from dependencies.